Poggers Report 31 (#27)
space-wizards/website-content: content/post/22-03-16-progress-report-31.md

+++
title = "Progress Report #31: Duck, Duck, Weh"
author = "ike709"
date = 2022-03-16
categories = [
    "Progress Report"
]
reddit = ""
thumbnail = "pr_31"
+++
It's been a minute since the last Progress Report, and there's been so many changes that we had to break this one up into two parts! This part covers January 2022.
<!--more-->

{{< newsection >}}
## Content
{{< newsection >}}
### Two-Parter
There's been a long list of contributions since the last progress report, so this edition just covers January 2022. The February 2022 edition will follow in a week or so.

{{< newsection >}}
### Game Wiki

> Space Station 14 now has a [wiki](https://wiki.spacestation14.io/wiki/Main_Page)! It's a one-stop shop for everything you need to know in order to get that greentext.

Reviewer: Perhaps mention that it is open to contributions? People can log in with their SS14 account.

> Space Station 14 now has a [wiki](https://wiki.spacestation14.io/wiki/Main_Page)! It's a one-stop shop for everything you need to know in order to get that greentext.

Author: ```suggestion
Space Station 14 now has a [wiki](https://wiki.spacestation14.io/wiki/Main_Page)! It's a one-stop shop for everything you need to know in order to get that greentext, and anyone with a Space Station 14 account is able to contribute.
```

{{< newsection >}}
### Rootbeer, Ice Cream, & Rootbeer Floats
*Contributed by Plasmaguy*

Nanotrasen has decided to reward its hardworking employees with a special treat! Swing by the bar for some newly stocked 100% Real Icecream* or a rootbeer float made with the highest quality rootbeer available (on the station)!

{{< imgw "images/post/pr_31/rootbeer" >}}

**Real Icecream™ is a registered trademark of Nanotrasen Inc. Individuals who have suffered a heart attack or have heart disease should not consume Real Icecream™*

{{< newsection >}}
### Lawyer & Librarian
*Contributed by Peptide90*

A foul evil has been detected aboard Space Station 14, it must be stopped before— Oh wait, it's just the lawyer. Better refresh yourselves on Space Law.

You can now also find the librarian hiding amongst the station's repository of knowledge and low-quality smut.

{{< newsection >}}
### Weh!
*Contributed by DogZeroX*

Weh!

{{< video-auto "/video/pr_31/weh.mp4" >}}

{{< newsection >}}
### Energy Swords
*Contributed by Mirino97*

Swish, swish, stab. It's a sword, dude, not a fighter jet.

{{< imgw "images/post/pr_31/esword" >}}

{{< newsection >}}
### Duck!

> *Contributed by Daemon*

Author: ```suggestion
*Contributed by Daemon and Mithrandalf*
```

Watch out for homemade flamethrowers!

{{< newsection >}}
### Misc. Changes
* Glass tables can now shatter. *Contributed by mirrorcult*
* Players now move slower. *Contributed by the maintainer team*
* Disposals pipes can now be constructed. *Contributed by DubiousDoggo*
* Uplinks can now buy Syndicate Smokes. *Contributed by Rane*

{{< newsection >}}
## Technical

{{< newsection >}}
### Multi-grid Stations & Multi-station Maps
*Contributed by mirrorcult*

Space Station 14 now supports having multiple stations or stations comprised of multiple grids. Gamemodes can also specify what number and combination of maps to load, opening the door for interesting new gamemodes like Station Versus Station.

{{< newsection >}}
### Server Whitelist
*Contributed by mirrorcult*

Server hosts can now enable a whitelist using the `whitelist.enabled` cvar. Only players added with the command `whitelistadd <username>` will be able to join, and they can be removed with `whitelistremove <username>`.

If you want to enable it mid-round, you can use the `kicknonwhitelisted` command to give every unwanted player the boot.

{{< newsection >}}
### Map Renderer

> *Contributed by DrSmugleaf and mirrorcult*

Author: ```suggestion
*Contributed by DrSmugleaf, moony, and mirrorcult*
```

> `Content.MapRenderer` has been added as a new project that can turn a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.

Author: ```suggestion
`Content.MapRenderer` has been added as a new project that can take a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.
```

> `Content.MapRenderer` has been added as a new project that can turn a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.

Author: ```suggestion
`Content.MapRenderer` has been added as a new project that can take a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.
```

{{< imgw "images/post/pr_31/bagelrender" >}}

{{< newsection >}}
## Credits
### Contributors Since Last Progress Report

> *354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Daemon, Delete69, ElectroJr, DubiousDoggo, Júlio César Ueti, Paul, ScalyChimp, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*

Reviewer 2: ```suggestion
*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Daemon, Delete69, ElectroJr, DubiousDoggo, Júlio César Ueti, Paul, ScalyChimp, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

> *354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*

Author: ```suggestion
*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, DubiousDoggo, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

> *354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, DubiousDoggo, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*

Reviewer 3: Duplicate names.

```suggestion
*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Daemon, Delete69, ElectroJr, DubiousDoggo, Júlio César Ueti, Paul, ScalyChimp, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

> *354 commits, authored by: metalgearsloth, Leon Friedrich, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*

Author: ```suggestion
*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

> *354 commits, authored by: metalgearsloth, Leon Friedrich, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, ZeroDayDaemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*

Author: ```suggestion
*354 commits, authored by: metalgearsloth, Leon Friedrich, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

> *354 commits, authored by: metalgearsloth, Leon Friedrich, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, ZeroDayDaemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*

Author: ```suggestion
*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, ZeroDayDaemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

### Patrons

> *Aaron Mell, Acruid, Alex Evgrashin, BubblegumBlue, Charlese2, Clyybber, Daemon, Delete69, Demetre Beroshvili, DrSmugleaf, E F R, ElectroJr, Fishfish458, Flipp Syder, DubiousDoggo, Julian Giebel, JustinTime, Júlio César Ueti, Kara D, Lamrr, LittleBuilderJane, Matz05, Michael Phillips, Moony, Morbo, Neith, Pancake, Paul, Peptide90, Pieter-Jan Briers, Plasmaguy, Rane, Sam Weaver, ScalyChimp, ShadowCommander, T-Stalker, TaralGit, Vera Aguilera Puerto, Willhelm53, WlarusFromDaSpace, Ygg01, Zymem, bloodrizer, keronshb, metalgearsloth, mirrorcult, pointer-to-null, router, sewerpig, wrexbe*

Reviewer 3: List of Patrons is wrong, I'm not one of them and Tomeno is missing for example.

Reviewer 3: Done in [`d33a7e6` (#27)](https://github.com/space-wizards/website-content/pull/27/commits/d33a7e6311c71bc1065c548a98b3b044ea77e021)

> *Aaron Mell, Acruid, Alex Evgrashin, BubblegumBlue, Charlese2, Clyybber, Daemon, Delete69, Demetre Beroshvili, DrSmugleaf, E F R, ElectroJr, Fishfish458, Flipp Syder, DubiousDoggo, Julian Giebel, JustinTime, Júlio César Ueti, Kara D, Lamrr, LittleBuilderJane, Matz05, Michael Phillips, Moony, Morbo, Neith, Pancake, Paul, Peptide90, Pieter-Jan Briers, Plasmaguy, Rane, Sam Weaver, ScalyChimp, ShadowCommander, T-Stalker, TaralGit, Vera Aguilera Puerto, Willhelm53, WlarusFromDaSpace, Ygg01, Zymem, bloodrizer, keronshb, metalgearsloth, mirrorcult, pointer-to-null, router, sewerpig, wrexbe*

Reviewer 3: Correct list:

```suggestion
*Phillip Inman, Zakanater 19, Jex N, Peter "Azmond" Newhouse, Seija, Tom Cruize, Corpus Inc., Hamcha, Oxyclean114, JustanAnus, NorthwoodTodd, Collin R Terrell, Cormos Lemming, grhmhome, Spiffyfishface, Callum Tubrett, Hydration, John Edward Hamilton Barchard, Wrexbe, Frege Beach, Austin Nelson, AquaDraco, Miniwoffer, hh, Nicholas Hillblom, Florian, Viridian, Daskata, Blaise M., Jack Rose, The Hateful Flesh, Will M., spinnermaster, Koyki, Valinov, Dan Griskardian, Robin Rottstock, Altana, Durp, Joshington Awesomahee, Eric VW, Evan Armstrong, Mono, Ethan Keller, DramaBuns, KrystalDisc, Darren Brady, Zandario, Anthony Fleck, vifs_vestige, Bobberunio, CPM311, ThatGuyGW, Farewell Fire, Daniel Thompson, Tomeno*
```

> *Aaron Mell, Acruid, Alex Evgrashin, BubblegumBlue, Charlese2, Clyybber, Daemon, Delete69, Demetre Beroshvili, DrSmugleaf, E F R, ElectroJr, Fishfish458, Flipp Syder, Jack Fox, Julian Giebel, JustinTime, Júlio César Ueti, Kara D, Lamrr, LittleBuilderJane, Matz05, Michael Phillips, Moony, Morbo, Neith, Pancake, Paul, Peptide90, Pieter-Jan Briers, Plasmaguy, Rane, Sam Weaver, ScalyChimp, ShadowCommander, T-Stalker, TaralGit, Vera Aguilera Puerto, Willhelm53, WlarusFromDaSpace, Ygg01, Zymem, bloodrizer, keronshb, metalgearsloth, mirrorcult, pointer-to-null, router, sewerpig, wrexbe*

Author: ```suggestion
*Aaron Mell, Acruid, Alex Evgrashin, BubblegumBlue, Charlese2, Clyybber, Daemon, Delete69, Demetre Beroshvili, DrSmugleaf, E F R, ElectroJr, Fishfish458, Flipp Syder, DubiousDoggo, Julian Giebel, JustinTime, Júlio César Ueti, Kara D, Lamrr, LittleBuilderJane, Matz05, Michael Phillips, Moony, Morbo, Neith, Pancake, Paul, Peptide90, Pieter-Jan Briers, Plasmaguy, Rane, Sam Weaver, ScalyChimp, ShadowCommander, T-Stalker, TaralGit, Vera Aguilera Puerto, Willhelm53, WlarusFromDaSpace, Ygg01, Zymem, bloodrizer, keronshb, metalgearsloth, mirrorcult, pointer-to-null, router, sewerpig, wrexbe*
```

> *Aaron Mell, Acruid, Alex Evgrashin, BubblegumBlue, Charlese2, Clyybber, Delete69, Demetre Beroshvili, DrSmugleaf, E F R, ElectroJr, Fishfish458, Flipp Syder, Jack Fox, Julian Giebel, JustinTime, Júlio César Ueti, Kara D, Lamrr, Leon Friedrich, LittleBuilderJane, Matz05, Michael Phillips, Moony, Morbo, Neith, Pancake, Paul, Peptide90, Pieter-Jan Briers, Plasmaguy, Rane, Sam Weaver, ScalyChimp, ShadowCommander, T-Stalker, TaralGit, Vera Aguilera Puerto, Willhelm53, WlarusFromDaSpace, Ygg01, ZeroDayDaemon, Zymem, bloodrizer, keronshb, metalgearsloth, mirrorcult, pointer-to-null, router, sewerpig, wrexbe*

Author: ```suggestion
*Aaron Mell, Acruid, Alex Evgrashin, BubblegumBlue, Charlese2, Clyybber, Daemon, Delete69, Demetre Beroshvili, DrSmugleaf, E F R, ElectroJr, Fishfish458, Flipp Syder, Jack Fox, Julian Giebel, JustinTime, Júlio César Ueti, Kara D, Lamrr, Leon Friedrich, LittleBuilderJane, Matz05, Michael Phillips, Moony, Morbo, Neith, Pancake, Paul, Peptide90, Pieter-Jan Briers, Plasmaguy, Rane, Sam Weaver, ScalyChimp, ShadowCommander, T-Stalker, TaralGit, Vera Aguilera Puerto, Willhelm53, WlarusFromDaSpace, Ygg01, Zymem, bloodrizer, keronshb, metalgearsloth, mirrorcult, pointer-to-null, router, sewerpig, wrexbe*
```
